code/obj/item/clothing/uniforms.dm

```diff
@@ -1114,11 +1114,11 @@
 		item_state = "darkred"
 
 	blue
-		icon_state = "scrub-b"
+		icon_state = "scrub-n"
```

Comment:
What's the `-n` here? 🤔

Not requiring any change as long as it works, just seems odd. "Navy?"

If so, follow-up PR in the near future to update the item path accordingly, please! Same for `-v` below (violet?)

Reply:
Okay so the scrubs were changed by flourish to have different icon names, being for navy and violet, with the filepath also being changed, but then I changed the filepath back so we didn't have to do map fixes, with the idea being that map fixes and filepath revert could be done after. All in all, it's intended to be like that